src/Network/Receive/kRO/RagexeRE_2015_11_04a.pm

```diff
@@ -17,9 +17,48 @@ package Network::Receive::kRO::RagexeRE_2015_11_04a;
 use strict;
 use base qw(Network::Receive::kRO::RagexeRE_2015_10_29a);
 
+# TODO: remove 'use Globals' from here, instead pass vars on
+use Globals qw(%config $bytesSent %packetDescriptions $enc_val1 $enc_val2 $char $masterServer $syncSync $accountID %timeout %talk $skillExchangeItem $net $rodexList $rodexWrite %universalCatalog %rpackets);
+use Log qw(message warning error debug);
+use Misc;
+use Utils;
+
 sub new {
 	my ($class) = @_;
-	return $class->SUPER::new(@_);
+	my $self = $class->SUPER::new(@_);
+	
+	my %packets = (
+		'0A14' => ['rodex_check_player', 'V v2', [qw(char_id class base_level)]],
```

Comment:
add to sakexe_0.pm ?

Reply:
I cannot guarantee the proposed packet structure to work on any other client than RagexeRE_2015_11_04a. Additionally it is mandatory to update the **subs** because they set the received values in $rodexWrite different than in **Receive.pm**. `@{$rodexWrite->{target}}{qw(char_id class base_level)} = @{$args}{qw(char_id class base_level)};`

Heres whats different on GGamerz RO:
sub rodex_open_write receive packet -> sends **charactername** back to client
sub rodex_check_player receive packet -> DOESNT send **charactername** back to client

sub rodex_send_mail send packet -> Had wrong title_len, body_len and assumed that {target} would contain {name}...